.prettierignore

```diff
@@ -0,0 +1,6 @@
+/dist
+/idea

```

Comment:
seem like we don't have these at the top level

Reply:
Correct :)